lib/bookingsync_portal.rb

```diff
@@ -21,6 +21,13 @@ module BookingsyncPortal
   mattr_accessor :portal_name
   @@portal_name = 'Portal'
 
+  # source name for use in bookings
+  mattr_accessor :source_name
+
+  # Allow to create a remote rental from this app.
+  mattr_accessor :create_remote_rental
+  @@create_remote_rental = false
```

Comment:
I think it would be the same with class instance variable, no? 

Class variables act extremely weird in some cases, I would not use them at all, it's not the case here, but e.g. modifying class variable from subclass modifies the class variable from the superclass, which might be quite surprising, with class instance variables it would not be the case, so by principle I would stay away from them.